README.md

```diff
@@ -168,16 +165,11 @@ psql "sslmode=require host=localhost port=5440 dbname=${DB_NAME} user=${DB_USER}
 
 <details>
 
-<summary>Bastion configuration</summary>
+Run 'az login'
```

Comment:
isn't there a minimum version for azure cli?

Reply:
Updated

Reply:
```suggestion
Run `az login`
```